### PR TITLE
Xp/requirements and auto installation

### PR DESCRIPTION
**Pull request by gregllong**
review: @jayich @aransfor 

@aransfor : Would be nice if you can modify the method ```_set_unix_environmental_variables()``` in [install_dependencies.py](https://github.com/CampbellGroup/common/blob/318b8d22200dad08165674bd1d5ee6041f3ab62d/install_dependencies.py#L117). Basically I'm not familiar with linux and I can't test on linux, so it would nice if you can complete that part.

### CODE REVIEW

**Comment by jayich** [comment]
lol regarding password.  LGTM.

**Comment by gregllong** [comment]
@jayich made two more small changes

File: install_dependencies.py [hunk 1]
```diff
@@ -3,6 +3,9 @@
 
 Also add LabRAD system environmental variables.
 
+The functions used to hoist to admin status is found in the following link:
```
Comment (jayich) [inline install_dependencies.py]: The functions used to host the admin status are found here:

File: README.md [hunk 2]
```diff
@@ -8,3 +8,5 @@ To install dependent modules, start a Terminal and run ```install_dependencies.p
 ```bash
 >>> python install_dependencies.py
 ```
+
+Please note that you might have to change the LABRADHOST environmental variable after installation. For instructions on how to do this, look up on CampbellGroup wiki.
```
Comment (jayich) [inline README.md]: You might have to change...   ...installation.  For instructions see this CampbellGroup wiki.

**Comment by jayich** [comment]
Small comments, otherwise LGTM.

**Comment by gregllong** [comment]
@aransfor can you also verify [this](https://github.com/CampbellGroup/common/pull/120/commits/27e0105ef641934ab2ebbf088b1308a879433bad) (and probably elaborate?)

**Comment by aransfor** [comment]
@gregllong So you guys need to upgrade opal kelly and see what the difficulties are. For linux the instructions you gave are sufficient. I Dont know the procedure or if we have the current OK api for windows. You guys should try and run the pulser2 server and when you can do this we delete the old pulser code

**Comment by gregllong** [comment]
@aransfor I guess for linux I'm more concerned about setting the environmental variables automatically during the setup. I found something that would temporarily change env-val for linux, but because I don't have access to a linux machine I could not test if that would permanently change the env-val, which is what we want. 

I'm tempted to merge this first, and make changes to setup for opal kelly after we update and test the new ok. And as for setting env-val for linux, I think the best way is probably wait till you have time, and then we can add that functionality in later. Thoughts?

**Comment by gregllong** [comment]
@aransfor bump

**Comment by aransfor** [comment]
Sorry for the lateness, I agree just merge it

**Comment by gregllong** [comment]
@aransfor No worries!

**Pull request merged**